CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## v0.1.1
```

Comment:
Is there a change in here that merits a release? I don't think any of these affect runtime code.

Reply:
No, but it fixes issues with our docs (see Documentation section). Cloud docs are generated against npm published versions, so we'll have to cut a patch to fix it.